client/source/js/modules/analysis/optimization-ctrl.js

```diff
@@ -446,6 +450,14 @@ define(['./module', 'angular', 'd3'], function (module, angular, d3) {
     $scope.validateOutcomeWeights = validateOutcomeWeights;
 
     /**
+     * Returns true if at least one chart is available
+     */
+    $scope.someGraphAvailable = function() {
+      return $scope.state.radarCharts || $scope.state.optimisationGraphs ||
+        $scope.state.financialGraphs || $scope.state.pieCharts;
+    };
+
```

Comment:
Nice comment and method.

Minor thing, I'd have named it `hasAvailableGraph`

Not a showstopper of course